src/Microsoft.AspNetCore.Sockets.Client/project.json

```diff
@@ -18,12 +18,20 @@
     "nowarn": [
       "CS1591"
     ],
-    "xmlDoc": true
+    "xmlDoc": true,
+    "compile": {
+      "include": [
```

Comment:
We could create a separate common project to share this code but it felt too much for these 3 file (one of which we hope will make it to the channels)